marlowe-dashboard-client/src/Template/View.purs

```diff
@@ -144,7 +144,8 @@ contractOverview contractTemplate =
             [ contractIcon contractTemplate.metaData.contractType
             , text $ contractTemplate.metaData.contractName <> " overview"
             ]
-        , p_ $ markdownToHTML contractTemplate.metaData.contractDescription
+        , p_ $ markdownToHTML contractTemplate.metaData.contractShortDescription
```

Comment:
```suggestion
        , p [ classNames [ "mb-4" ] ] $ markdownToHTML contractTemplate.metaData.contractShortDescription
```

Reply:
That is great, thanks. I didn't know the suggested change feature too, nice :)